app/routes/user/endorsements.server.ts

```diff
@@ -87,7 +88,7 @@ export class EndorsementsServer {
    *   - User `endorserSub` does not have an email attribute
    *   - There is an existing endorsement with the same requester and submitter sub
    */
-  async createEndorsementRequest(endorserSub: string) {
+  async createEndorsementRequest(endorserSub: string, note?: string) {
```

Comment:
The DynamoDB API calls will probably fail if `note` is `undefined`, which is permitted by the type signature. DynamoDB generally seems to support `null` but not `undefined`.